src/openforms/registrations/contrib/objects_api/tests/test_templatetags.py

```diff
@@ -63,6 +65,74 @@ def test_render_json_summary(self):
 
         self.assertEqual(rendered, expected)
 
+    @override_settings(ESCAPE_REGISTRATION_OUTPUT=False)
+    def test_render_json_summary_doesnt_escape_html_when_disabled(self):
+        SubmissionStepFactory.create(
+            form_step__form=self.form,
+            form_step__form_definition__slug="json-summery-step",
+            form_step__form_definition__configuration={
+                "display": "form",
+                "components": [
+                    {
+                        "key": "voornaam",
+                        "type": "textfield",
+                        "registration": {
+                            "attribute": RegistrationAttribute.initiator_voornamen,
+                        },
+                    }
+                ],
+            },
+            submission=self.submission,
+            data={
+                "voornaam": "<script>alert();</script>",
+            },
+        )
+
+        rendered = render_from_string(
+            "{% json_summary %}",
+            context={"_submission": self.submission},
+            backend=openforms_backend,
+            disable_autoescape=True,
+        )
+
+        expected = '{"json-summery-step": {"voornaam": "<script>alert();</script>"}}'
```

Comment:
with the above code suggestion, the slug of the form is changed a bit:

```suggestion
        expected = '{"formstep-slug": {"voornaam": "<script>alert();</script>"}}'
```